1 ** Message: console message: undefined @0: TypeError: 'null' is not an object
2 /home/justzx/workplace/openyoudao/webshot.py:53: Warning: g_main_context_push_thread_default: assertion `acquired_context' failed
3 Unhandled exception in thread started by <function gettext at 0x1ea2c08>
Traceback (most recent call last):
	  File "main.py", line 89, in gettext
	      record_dpy.record_enable_context(ctx,record_callback)
	    File "/usr/lib/pymodules/python2.7/Xlib/ext/record.py", line 238, in enable_context
	      context = context)
	    File "/usr/lib/pymodules/python2.7/Xlib/ext/record.py", line 215, in __init__
	      apply(rq.ReplyRequest.__init__, (self, ) + args, keys)
	    File "/usr/lib/pymodules/python2.7/Xlib/protocol/rq.py", line 1428, in __init__
	      self.reply()
	    File "/usr/lib/pymodules/python2.7/Xlib/protocol/rq.py", line 1440, in reply
	      self._display.send_and_recv(request = self._serial)
	    File "/usr/lib/pymodules/python2.7/Xlib/protocol/display.py", line 547, in send_and_recv
	      gotreq = self.parse_response(request)
	    File "/usr/lib/pymodules/python2.7/Xlib/protocol/display.py", line 634, in parse_response
	      gotreq = self.parse_request_response(request) or gotreq
	    File "/usr/lib/pymodules/python2.7/Xlib/protocol/display.py", line 720, in parse_request_response
	      req._parse_response(self.data_recv[:self.request_length])
	    File "/usr/lib/pymodules/python2.7/Xlib/ext/record.py", line 219, in _parse_response
	      self._callback(r)
	    File "main.py", line 38, in record_callback
	      event, data = rq.EventField(None).parse_binary_value(data, record_dpy.display, None, None)
	  AttributeError: 'NoneType' object has no attribute 'EventField'

